lib/fathead/caniuse/redirects.txt

```diff
@@ -0,0 +1,976 @@
+svg filters,svg-filters
+svg filters,svg-filters
+svg filters,svg-filters
```

Comment:
@moollaza I didn't know the format of how to add redirects in output.txt, when i was making it, but fixed that now. Hope it's the correct format. 

and the 3 of every redirect was a bug in how i made the redirects list. but, it's fixed now.